From 9b529c2f911e9a7ad25586d0d4a8db8bc5a7b2e5 Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Thu, 5 Aug 2021 22:44:48 +0000
Subject: [PATCH] Revert sourcelink submodule workaround

aspnetcore will fail the offline tarball build because of the sourcelink workaround being removed here.
To fix this, a sourcelink patch was added to address the issue.
---
 eng/SourceBuild.props | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/eng/SourceBuild.props b/eng/SourceBuild.props
index 3dbfe2ce7e..403b91f976 100644
--- a/eng/SourceBuild.props
+++ b/eng/SourceBuild.props
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <GitHubRepositoryName>aspnetcore</GitHubRepositoryName>
     <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
-    <CloneSubmodulesToInnerSourceBuildRepo>false</CloneSubmodulesToInnerSourceBuildRepo>
   </PropertyGroup>
 
   <Target Name="PrepareGlobalJsonForSourceBuild"
@@ -13,19 +12,6 @@
       WorkingDirectory="$(InnerSourceBuildRepoRoot)" />
   </Target>
 
-  <!--
-    Init submodules - temporarary workaround for https://github.com/dotnet/sourcelink/pull/653
-  -->
-  <Target Name="InitSubmodules"
-          DependsOnTargets="PrepareInnerSourceBuildRepoRoot"
-          BeforeTargets="RunInnerSourceBuildCommand">
-
-    <Exec
-      Command="git submodule update --init --recursive"
-      WorkingDirectory="$(InnerSourceBuildRepoRoot)"
-      EnvironmentVariables="@(InnerBuildEnv)" />
-  </Target>
-
   <!--
     Remove inner source .globalconfig file as both the inner and outer config files get loaded and cause a conflict.
     Leaving the inner will cause all conflicting settings to be ignored.
-- 
2.29.2

